providers/keycloak_oidc.go

```diff
@@ -21,6 +21,10 @@ func NewKeycloakOIDCProvider(p *ProviderData, opts options.KeycloakOptions) *Key
 		name: keycloakOIDCProviderName,
 	})
 
+	if opts.BackendLogoutURL != "" {
+		p.BackendLogoutURL = opts.BackendLogoutURL
+	}
```

Comment:
I don't think the check is necessary.
```suggestion
	p.BackendLogoutURL = opts.BackendLogoutURL
```